farm-boosted-rewards-subscriber/src/subscriber_config.rs

```diff
@@ -1,14 +1,28 @@
 multiversx_sc::imports!();
 multiversx_sc::derive_imports!();
 
-use common_structs::UniquePayments;
 use config::ProxyTrait as _;
 use subscription_fee::subtract_payments::Epoch;
 
 pub type Percentage = u32;
 pub const TOTAL_PERCENTAGE: Percentage = 10_000;
 pub const EPOCHS_IN_WEEK: u64 = 7;
 
+#[derive(Default, TypeAbi, TopEncode, TopDecode)]
+pub struct UserLastPayment {
+    pub service_index: usize,
+    pub epoch: Epoch,
+}
+
+impl UserLastPayment {
+    pub fn new(service_index: usize, epoch: Epoch) -> Self {
```

Comment:
I'd remove this method, just initialize in-line if there are no additional conditions for the arguments.

Reply:
Done.